# Listy

## [Listy](../../Teoria/Typy%20danych/Lista.md)

## Zadania

In [4]:
# Zadanie

cities = ['New York', 'London', 'Istanbul', 'Seoul']

# Dodaj na koniec listy 'Sydney'.
cities.append('Sydney')
print(cities)

['New York', 'London', 'Istanbul', 'Seoul', 'Sydney']


In [9]:
# Zadanie

repeat_list = [1, 2, 1, 3, 4, 1, 3, 5, 1, 2]

# Zlicz wystąpienia elementu 1

print(repeat_list.count(1))

4


In [14]:
# Zadanie
text = 'Programowanie w języku Python'
# Enter your solution here

text = text.replace('ę', 'e').replace(' ', '').lower()
text = set(text)
text = sorted(text)

text = text[0:10]
print(text)

['a', 'e', 'g', 'h', 'i', 'j', 'k', 'm', 'n', 'o']


In [16]:
# Zadanie
list_cities = ['New York', 'London', 'Istanbul', 'Seoul']

# Dodaj warszawe na poczatek listy

list_cities.insert(0, 'Warszawa')
list_cities.remove('Istanbul')
print(list_cities)

['Warszawa', 'New York', 'London', 'Seoul']


In [17]:
# Zadanie
list_techs = ['HTML', 'CSS', 'JavaScript', 'Python', 'PHP']

print(sorted(list_techs))

['CSS', 'HTML', 'JavaScript', 'PHP', 'Python']


In [21]:
# Zadanie

list_words = ['apple', 'orange', 'banana', 'pineapple', 'pear']

# wynik = "#apple#orange#banana#pineapple#pear"

print('#' + '#'.join(list_words))

#apple#orange#banana#pineapple#pear


In [30]:
scorers = [27, 8, 15, 2, 9, 10, 21, 4, 20, 5]
# Enter your solution here
sorted_list = sorted(scorers, reverse=True)

print(sorted_list[:3])

[27, 21, 20]


In [1]:
players = ['Ronaldo', 'Messi', 'Cristiano', 'Neymar']
scores = [27, 18, 22, 15]

new_players = ['Mbappe', 'Ibrahimovic', 'Hazard']
new_scores = [27, 18]

# players.extend(new_players)
# scores.extend(new_scores)

print(players+new_players)
print(scores+new_scores)

['Ronaldo', 'Messi', 'Cristiano', 'Neymar', 'Mbappe', 'Ibrahimovic', 'Hazard']
[27, 18, 22, 15, 27, 18]


['Caesar Salad', 9.99, ['romaine lettuce', 'croutons', 'parmesan cheese', 'Caesar dressing'], 'olive oil']


"['Ca[42 chars]outons', 'parmesan cheese', 'Caesar dressing', 'olive oil']]\n" != "['Ca[42 chars]outons', 'parmesan cheese', 'Caesar dressing'], 'olive oil']\n"
- ['Caesar Salad', 9.99, ['romaine lettuce', 'croutons', 'parmesan cheese', 'Caesar dressing', 'olive oil']]
?                                                                                                          -
+ ['Caesar Salad', 9.99, ['romaine lettuce', 'croutons', 'parmesan cheese', 'Caesar dressing'], 'olive oil']
?  